.gitignore

```diff
@@ -12,3 +12,5 @@ Pods/
 
 Carthage/Build
 Carthage/Checkouts
+/documentation
```

Comment:
Right, but not in this repo. We should probably check these in on a separate repo similar to how we handle https://github.com/mapbox/ios-sdk